docs/conops.md

```diff
@@ -0,0 +1,146 @@
+
+See the [High-Level CONOPS](https://github.com/Arrow-air/se-services/blob/develop/docs/conops.md).
+
+## Business Model Discussion
+
+### Vertical Integration
+The current assumed business model is  bearing the cost of building and operating aircraft.
+
+Under this business model, the motivation of pricing is to recoup the CapEx investments in building aircraft and the OpEx in operating the fleet. While Arrow may benefit from the simplicity of the business model and enjoy the superior quality control from a vertically integrated business, the upfront capital and talent investments may result in a protracted pre-revenue phase.
+
+More, the OEM space is already crowded with mature players like [Lilium](https://lilium.com/), [Joby](https://www.jobyaviation.com/), [Elroy Air](https://elroyair.com/), [Zipline](https://www.flyzipline.com/), etc., many of them have already had commercial operations ([here](https://www.pfizer.com/news/press-release/press-release-detail/zipline-pfizer-and-biontech-collaboration-paves-way) and [here](https://aerospaceamerica.aiaa.org/elroy-fedex-tests-cargo-delivery/)). The Services team believes that competing in a traditional, CapEx- and operation-heavy business segment is challenging given the long development cycle and the dynamics of commoditization.
+
+Last but not least, certifications with different government agencies around the world can be difficult to navigate and require significant legal resources and even lobbying efforts. Diverting our limited resources and contributors' time on bureaucratic processes may drag down the progress of the entire organization, especially given that Arrow operates on an anonymity-first principle, which can draw extra scrutiny from aviation regulatory bodies.
+
+### Marketplace Platform
+An alternative business model would be a marketplace platform like that of Uber. Arrow would solicit bids from OEM players to supply their fleet to our network, and each OEM player would have the discretion to submit a bidding for their vehicles for particular trips.
+
+For example, if a customer wants to ship a 10-kg cargo from San Francisco to San Jose departing April 10th, Arrow would present a list of options. So, the customer may see Elroy, Joby, and Zipline vehicles available for this particular itinerary but at different pricing points:
+
+*(Illustrative purpose only)*
+* Joby Cargo: $47.2
+* Lilium Cargo: $50.1
+* Zipline Cargo: $51.3
+
+All else being equal, the customer would naturally pick the Joby option because it is the cheapest. This behavior will incentivize OEMs to innovate in order to compete with other players on price.
+
+The innovation can come from hardware (advanced manufacturing, superior procurement management, economic design, etc), or from algorithm. Some well-funded players may not care about recouping CapEx, so they would develop algorithms to analyze the supply and demand of the market, pricing of other services, etc., to programmatically price their trips.
```

Comment:
Mention the API explicitly here, "programmatically" implies it

Then the first line of the next sentence follows better

Reply:
[caa132d](https://github.com/Arrow-air/svc-pricing/pull/3/commits/caa132d62cbff9e89dee9650443a7a8e39d9d466)